src/gt4py/backend/templates/stencil_module.py.in

```diff
@@ -186,7 +186,12 @@ class {{ class_name }}(StencilObject):
                         + stencil_info["run_cpp_time"]
                     )
 
-    def run(self, _domain_, _origin_, exec_info, *, {{- field_names|join(", ") -}}, {{- param_names|join(", ") -}}):
+    def run(self, _domain_, _origin_, exec_info,
+        {{- "*," if field_names|length > 0 or param_names|length > 0 -}}
+        {{- field_names|join(", ") -}}
+        {{- "," if field_names|length > 0 -}}
+        {{- param_names|join(", ") -}}
+    ):
```

Comment:
This is not needed after reverting `base.py` changes.

Reply:
Although no longer needed, it could be worth to keep these changes as they improve the fault-tolerance of the template.

Reply:
Not really, if unused fields are not removed, there will always at least 1 field, otherwise the pipeline won't reach this point, so the checks here are redundant and complicate a bit the template.

```suggestion
    def run(self, _domain_, _origin_, exec_info, *, {{- field_names|join(", ") -}}, {{- param_names|join(", ") -}}):
```